library/ix-dev/community/overseerr/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+name: overseerr
+description: Overseerr is a free and open source software application for managing requests for your media library.
+annotations:
+  title: Overseerr
+type: application
+version: 1.32.5
+apiVersion: v2
+kubeVersion: '>=1.16.0-0'
```

Comment:
```suggestion
version: 1.0.0
apiVersion: v2
apppVersion: '1.32.5'
kubeVersion: '>=1.16.0-0'
```

Reply:
version == chart version
appVersion == the container version (in this case overseerr)